<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTransactionPoolThingEdit</class>
 <widget class="QDialog" name="FTransactionPoolThingEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transactions pool edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/hacc.qrc">
    <normaloff>:/items/transaction-pool.png</normaloff>:/items/transaction-pool.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Date and time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dtDateTime">
         <property name="currentSection">
          <enum>QDateTimeEdit::DaySection</enum>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lBuyer">
         <property name="text">
          <string>Buyer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ui::selector::WContractorAccountSelectButton" name="pwBuyerAccount" native="true">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lSeller">
         <property name="text">
          <string>Seller</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ui::selector::WContractorAccountSelectButton" name="pwSellerAccount" native="true">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label4">
     <property name="text">
      <string>Transactions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::selector::WContractorAccountSelectButton</class>
   <extends>QWidget</extends>
   <header>wcontractoraccountselectbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dtDateTime</tabstop>
  <tabstop>pwBuyerAccount</tabstop>
  <tabstop>pwSellerAccount</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FTransactionPoolThingEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FTransactionPoolThingEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
